File: class_7.0_project_demo/src/main/java/com/model/SignUp.java
package com.model;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
public class SignUp {

	
	private String fullname;
	@Id
	private String email;
	private String pwd;
	public String getFullname() {
		return fullname;
	}
	public void setFullname(String fullname) {
		this.fullname = fullname;
	}
	public String getEmail() {
		return email;
	}
	public void setEmail(String email) {
		this.email = email;
	}
	public String getPwd() {
		return pwd;
	}
	public void setPwd(String pwd) {
		this.pwd = pwd;
	}
	
	
}

File: class_7.0_project_demo/src/main/webapp/WEB-INF/static/js/signup.js
$(document).ready(function(){
    $("#signup").click(function () { 
        var fullname=$("#fullname").val();
        var email=$("#email").val();
        var pwd=$("#pwd").val();

        $.post("add",{
            fullname:fullname,
            email:email,
            pwd:pwd,
        },function(response){
            alert(response);
        });
    });
});